package.json

```diff
@@ -17,7 +17,7 @@
     "@redhat-cloud-services/frontend-components-utilities": "^4.0.0",
     "@redhat-cloud-services/insights-common-typescript": "6.1.1",
     "@redhat-cloud-services/rbac-client": "1.2.2",
-    "@redhat-cloud-services/types": "^1.0.14",
+    "@redhat-cloud-services/types": "^1.0.23",
```

Comment:
Confirmed the drawer API was published in master via `1.0.23` and pf6 via `2.0.0-beta.2` of the types package (we need both master and PF6 versions as stage is running PF6 until we merge PF6 back into master):
master: https://github.com/RedHatInsights/frontend-components/commit/34823fc29309314a7f216e72eb99205dd25d2b8c
pf6: https://github.com/RedHatInsights/frontend-components/commit/fadc67b7369be604f169e01d2f8af979d2b1a165